test/libyul/yulInterpreterTests/external_call_to_self.yul

```diff
@@ -14,7 +14,6 @@
 // ----
 // Trace:
 //   CALL(153, 0x11111111, 0, 64, 32, 256, 32)
-//   RETURN(0, 0)
```

Comment:
Why remove all these traces? They technically don't access memory, but the actual return (resp. log/create) still happens...

Reply:
> Why remove all these traces?

Mostly because the optimizer converts `return(12345, 0)` to `return(0, 0)` and consequently diverge traces.

Reply:
Hm... we could print ``RETURN(0, 0)`` for all empty returns in the trace, maybe :-)?
The danger in just removing the traces is that we might miss issues where some return is actually missing, isn't it? Returning with an empty return vs not returning at all is a huge difference...

Reply:
Good point, I'll update the PR accordingly

Reply:
Updated.